packages/application-extension/schema/top-bar.json

```diff
@@ -0,0 +1,75 @@
+{
+  "title": "Top Bar",
+  "description": "Top Bar settings.",
+  "jupyter.lab.toolbars": {
+    "TopBar": [
+      {
+        "name": "spacer",
+        "type": "spacer",
+        "rank": 50
+      }
```

Comment:
I would not add the spacer here. And prefer to register a new widget in the collaboration-extension to the toolbar that create a spacer to be added with the user menu only in collaborative mode. This should help not impacting the UI in non-collaborative mode.

```suggestion
```

Reply:
> only in collaborative mode

Right that would be a good compromise.